frontend/src/metabase/actions/components/ActionForm/ActionForm.unit.spec.tsx

```diff
@@ -338,13 +336,11 @@ describe("Actions > ActionForm", () => {
 
       userEvent.click(screen.getByRole("button", { name: "Save" }));
 
-      expect(
-        await screen.findByText(/this field is required/i),
-      ).toBeInTheDocument();
+      expect(await screen.findByText(/required/i)).toBeInTheDocument();
       expect(onSubmit).not.toHaveBeenCalled();
     });
 
-    it("disables form submission when no fields are changed", async () => {
+    it("allows form submission when all fields are optional", async () => {
```

Comment:
Should we also add a test checking that submit is disabled if there's at least one parameter without a value yet?